packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1412,6 +1412,11 @@ const schema: SchemaType<DbUser> = {
     ...notificationTypeSettingsField({ channel: "both" }),
     hidden: !dialoguesEnabled,
   },
+  notificationNewDialogueChecks: {
+    label: "You have new people interested in dialogue-ing with you",
+    ...notificationTypeSettingsField({ batchingFrequency: "daily" }),
```

Comment:
if `batchingFrequency` is set to daily, an hourly cron job won't do very much.  probably this should just not have a batching frequency?

Reply:
(or, rather, it should be `realtime`, which I think is the default)

Reply:
I was thinking most people would have it be "daily", but if you wanted "realtime" you would get hourly

Reply:
Sure, just keep in mind that most people never change any settings like this (and may not even know that they can)